modules/microvm/virtualization/microvm/netvm.nix

```diff
@@ -62,13 +62,19 @@
 
         microvm = {
           optimize.enable = true;
-          hypervisor = "qemu";
+          hypervisor = {
+            # Use the same machine type as the host
+            x86_64-linux = "cloud-hypervisor";
+            aarch64-linux = "qemu";
```

Comment:
why having a different vmm per architecture?

Reply:
The cloud-hypervisor PCI support on AArch64 is limited in functionality, and does not work with our wifi adaptor. Further, the PCI passing on AArch64 would only work in UEFI mode, which microvm does not support; and would probably be slower.